app/src/main/java/com/framgia/forder/data/source/remote/api/request/RegisterShopRequest.java

```diff
@@ -1,8 +1,10 @@
 package com.framgia.forder.data.source.remote.api.request;
 
-import com.framgia.forder.data.model.Shop;
+import android.net.Uri;
```

Comment:
ERROR: Unused import: android.net.Uri. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports